tendrl/node_agent/discovery/sds/plugins/discover_gluster_sds_details.py

```diff
@@ -91,11 +91,31 @@ def discover_storage_system(self):
                 version_det = lines[0].split(
                     'glusterfs-server-'
                 )[-1]
-                ret_val['pkg_version'] = "%s.%s.%s" % (
-                    version_det.split('.')[0],
-                    version_det.split('.')[1],
-                    version_det.split('.')[2]
-                )
-                ret_val['pkg_name'] = "gluster"
+                maj_ver = version_det.split('.')[0]
+                min_ver = version_det.split('.')[1]
+                rel = version_det.split('.')[2]
+                build_no = None
+                if '-' in rel:
+                    build_no = rel.split('-')[1]
+                    rel = rel.split('-')[0]
+                # Check for rpm version. if its something less than 3.8.4-52
+                # its not supported version
+                if int(maj_ver) == 3 and int(min_ver) == 8 and \
+                    int(rel) == 4 and build_no is not None and \
+                    int(build_no) < 52:
+                    ret_val['pkg_name'] = 'RHGS'
+                    ret_val['pkg_version'] = 'un-supported'
+                elif int(maj_ver) == 3 and int(min_ver) == 8 and \
+                    int(rel) == 4 and build_no in not None and \
```

Comment:
The failing [builds](https://ci.centos.org/view/tendrl-PR/job/tendrl-build-1-pr-node-agent/259/console) are because of following syntax issue:
```
File "/usr/lib/python2.7/site-packages/tendrl/node_agent/discovery/sds/plugins/discover_gluster_sds_details.py", line 109
    int(rel) == 4 and build_no in not None and \
```